classes/01class/exercises/c01-aws02/ramiray/Readme.md

```diff
@@ -0,0 +1,42 @@
+1. Created s3 bucket using following command. 
+
+```
+aws s3 mb s3://ramyadevopstest --region ap-southeast-2
+make_bucket: s3://ramyadevopstest
+
+```
```

Comment:
Hey Rami,
this one is from another exercise. Did you forget to checkout from master before submitting the new one? Happens to most of us! 😄  Let us know if you need any help fixing this issue

Reply:
Hi Christina,

I have updated the c01-aws02 file. 

Thanks,
Ramya